crates/solidity/outputs/cargo/crate/Cargo.toml

```diff
@@ -25,16 +25,27 @@ categories = [
   "text-editors",
 ]
 
+[[bin]]
```

Comment:
I suggest adding a changeset to this PR, to announce the introduction of the new feature. You can do so by running `changeset`, selecting a `minor` bump, and a message, for example:

> introduce a `cli` Cargo feature to compile the CLI binary

[Click here to learn what changesets are, and how to add one](https://github.com/changesets/changesets/blob/main/docs/adding-a-changeset.md).